lib/form/html.js

```diff
@@ -76,11 +77,15 @@ const addRequired = (html, input, index) => {
         required: input.inputs[inp].required,
         name: `${input.id}--${inp}`,
         id: input.inputs[inp].id,
+        type: input.inputs[inp].type,
       });
     });
   }
 
   inputs.map(inp => {
+    if (input.type === 'checkbox' || input.type === 'radio') {
+      return;
+    }
```

Comment:
Can you throw a line between the two if statements and add some comments pleaes?